pages/k8s/release-notes.md

```diff
@@ -14,6 +14,50 @@ toc: False
 ---
 
 
+# NAME OF RELEASE
+
+### DATE - BUNDLE REVISION WITH LINK
+
+## What's new
+
+- Calico 3.x support
+
+The Calico and Canal charms have been updated to install Calico 3.6.1 by
+default. For users currently running Calico 2.x, the next time you upgrade your
+Calico or Canal charm, the charm will automatically upgrade to Calico 3.6.1
+with no user intervention required.
+
+The Calico charm's `ipip` config option has been changed from a boolean to a
+string to allow for the addition of a new mode. This change is illustrated in
+the table below:
+
+| New value     | Old value         | Description                                           |
+|---------------|-------------------|-------------------------------------------------------|
+| "Never"       | false             | Never use IPIP encapsulation. (The default)           |
+| "Always"      | true              | Always use IPIP encapsulation.                        |
+| "CrossSubnet" | \<Not supported\> | Only use IPIP encapsulation for cross-subnet traffic. |
+
+- Calico BGP support
+
+Several new config options have been added to the Calico charm to support BGP
+functionality within Calico. These additions make it possible to configure
+external peerings, route reflectors, and complex IP pool configurations.
```

Comment:
I'd like to add a link here to the [Calico BGP docs](https://github.com/charmed-kubernetes/kubernetes-docs/pull/202), but the CNI pages aren't in yet. @evilnick any advice?

Reply:
You can add the link based on where they will be, because I intend to merge the CNI stuff before release anyhow